src/lsp/client.ts

```diff
@@ -81,4 +81,8 @@ export class SonarLintExtendedLanguageClient extends LanguageClient {
   getSuggestedBinding(configScopeId:string, connectionId: string): Promise<protocol.GetSuggestedBindingResponse>{
     return this.sendRequest(protocol.GetSuggestedBinding.type, {configScopeId, connectionId});
   }
+
+  checkLocalDetectionSupported(folderUri: string): Promise<protocol.CheckLocalDetectionSupportedResponse> {
```

Comment:
I believe this should mention that this checks for the local detection of hotspots.